src/OrderFormAcceptanceTests.TestData/Helpers/OrderItemHelper.cs

```diff
@@ -67,6 +67,17 @@ public static async Task<OrderItem> AddRecipientToOrderItem(OrderItem orderItem,
             return orderItem;
         }
 
+        public static async Task<int> GetNumberOfPricingUnitsForItemAsync(string itemId, string connectionString)
+        {
+            var query = @"SELECT COUNT(*)
+                          FROM [buyingcatalogue].[dbo].[CataloguePrice]
+                          WHERE CatalogueItemId = @itemId;";
```

Comment:
I'd recommend omitting the DB name as it could cause problems if running against an environment with a differently named DB.

You can omit the square brackets too.